StabilityClient.Net/StabilityClient.Net/Protos/generation.proto

```diff
@@ -404,6 +404,13 @@ message Answer {
   repeated Artifact artifacts = 7;
 }
 
+// An AnswerBatch is a set of Answers. It can represent one or several completed
+// requests, which may be sent to the client in a single response.
+message AnswerBatch {
```

Comment:
maybe "BatchedAnswer" is a better name? or this is not your code?

Reply:
This is automatic update from StabilitySdk